common/changes/@itwin/core-frontend/Frank-deprecation-quantity_2024-09-24-21-17.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Refactored parseToQuantityValue to parseQuantityString in core frontend tests",
```

```suggestion
      "comment": "",
```

Change logs are for users. Users don't care about tests.